buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -324,6 +324,11 @@ def get_summary_table(identifier) {
         return ''
     }
 
+    if (!TESTS_TARGETS) {
```

Comment:
I'm confused why this is here if we set it earlier?

Reply:
TESTS_TARGETS is set only for pipelines not for the wrappers.